### PR TITLE
Create registration test with negative scenario, registration page

### PR DESCRIPTION
**Pull request by Wolmeright**

### CODE REVIEW

File: src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import BasePageAndTestInit.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class NewBalanceLogInAndRegistrationPage extends BasePage {
+    public NewBalanceLogInAndRegistrationPage(WebDriver driver) {
+        super(driver);
+    }
```
Comment (AndriiMyronenko418) [inline src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java:12]: add empty lines before and after the constructor (9&10 and 12&13 lines)

File: src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        return $(INVALID_EMAIL_MESSAGE);
+    }
+    public WebElement fillInTheField(){
+        return $(FILL_THE_FIELD_MESSAGE);
+    }
+    public WebElement shortPasswordMessage(){
+        return $(SHORT_PASSWORD);
+    }
+    public WebElement passwordsNotSameMessage(){
+        return $ (PASSWORDS_NOT_SAME);
```
Comment (AndriiMyronenko418) [inline src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java:60]: add empty lines between methods

**Review by AndriiMyronenko418 — CHANGES_REQUESTED** [review]

File: src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java (hunk 1 above)
Comment (Wolmeright) [reply src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java:13]: fixed

File: src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java (hunk 2 above)
Comment (Wolmeright) [reply src/test/java/NewBalance/Pages/NewBalanceLogInAndRegistrationPage.java:73]: fixed

**Review by Tsibatyuk — APPROVED** [review]

**Review by AndriiMyronenko418 — APPROVED** [review]

**Pull request merged**